packages/components/src/components/toggle-switch/toggle-switch.jsx

```diff
@@ -2,27 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-class ToggleSwitch extends React.Component {
-    render() {
-        return (
-            <>
-                <input
-                    className={classNames('dc-toggle-switch', {}, this.props.className)}
-                    id={this.props.id}
-                    type='checkbox'
-                    checked={this.props.is_enabled}
-                    onChange={this.props.handleToggle}
-                />
-                <label
-                    className={classNames('dc-toggle-switch__label', {}, this.props.classNameLabel)}
-                    htmlFor={this.props.id}
-                >
-                    <span className={classNames('dc-toggle-switch__button', {}, this.props.classNameButton)} />
-                </label>
-            </>
-        );
-    }
-}
+const ToggleSwitch = ({ className, id, is_enabled, handleToggle, classNameLabel, classNameButton }) => (
```

Comment:
Can we have these props sorted alphabetically since we do it for other components?